### DISCUSSION TITLE
fail on the last 4 phase

### DISCUSSION THREAD

**Discussion by geep2112** [Chia-Network/chia-blockchain#6828] [Support]
fail on the last 4 phase when copying the 4 plots, the program rebooted itself and is connecting to the wallet more than an hour, what should I do?

**Comment by SoMoney** [comment]
Sounds like you have Wallet DB corruption.

1. STOP CHIA
2. Delete all the files lin
     `C:\Users\YOUR_USERNAME_GOES_HERE\.chia\mainnet\wallet\db`
3. START CHIA

**Reply by geep2112** [reply]
unfortunately, I had to restart my PC, lose 4 plots and everything works again, the main sadness is the loss of 4 plots, the machine had been worked worked all night...(

**Comment by BrandtH22** [comment]
@geep2112 , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
